### PR TITLE
220 Documentation for setting up debugging in Jet Brains IDE.

### PR DESCRIPTION
**Pull request by nschoenbrot**
# Purpose
Documentation for debugging with a Jet Brains IDE. 

# Changes
Adds a md file and screenshots. 

https://github.com/ethyca/fidesops/blob/220_jetbrains_debug_doc/docs/fidesops/docs/development/jetbrains_debugging.md

# Ticket
#220

### CODE REVIEW

**Review by seanpreston — CHANGES_REQUESTED** [review]
Awesome start! We need a bit more prose contextualising each step, especially in `Connect to Docker daemon` and `Configure Python Remote Interpreter`.

**Comment by nschoenbrot** [comment]
> Awesome start! We need a bit more prose contextualising each step, especially in `Connect to Docker daemon` and `Configure Python Remote Interpreter`.

Pushed up a commit with more details added to the instructions.

File: docs/fidesops/docs/development/jetbrains_debugging.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+Set up a Run/Debug Configuration so that breakpoints can be hit in the Fidesops sourcecode. 
+
+Go to: **Run/Debug Configurations** -> **+** -> **Python**
+
+- To debug Fidesops, debug the `<path on your machine>/src/fidesops/main.py` script
+- Make sure to select **Use specified interpreter** set the Remote Python Docker Compose *(created in the previous section)*
+- Add `FIDESOPS_CONFIG_PATH=/fidesops` to **Environment variables**
+
+See screenshot bellow:
```
Comment (seanpreston) [inline docs/fidesops/docs/development/jetbrains_debugging.md:53]: Typo: below

**Comment by seanpreston** [comment]
Thanks @nschoenbrot, just that one typo then it's good to merge

File: docs/fidesops/docs/development/jetbrains_debugging.md (hunk 1 above)
Comment (nschoenbrot) [reply docs/fidesops/docs/development/jetbrains_debugging.md]: Thanks, I pushed up the fix.

**Comment by nschoenbrot** [comment]
> Thanks @nschoenbrot, just that one typo then it's good to merge

Thanks, I made the fix.

**Review by seanpreston — APPROVED** [review]

**Pull request merged**